src/playground/playground-pug.ts

```diff
@@ -0,0 +1,25 @@
+import configurations from '../configurations';
+import '../extensions';
+// TODO change logger to just use console in this case
+import logger from '../utilities/logger';
+import '../global-error-handlers';
+import * as pug from 'pug';
+
+const enabledMarker = new Array(20).join('*');
+const disabledMarker = new Array(20).join('#');
+if (configurations.email.enabled) {
+    logger.info(`${enabledMarker} EMAIL ENABLED ${enabledMarker}`);
+} else {
+    logger.info(`${disabledMarker} EMAIL DISABLED ${disabledMarker}`);
+}
+
+// Pug playground doesn't specifically need the database.
+// import { sync } from '../database';
+
+
+(async (): Promise<void> => {
```

Comment:
This is a little different than how I normally do the playgrounds, if I'm going to be totally honest I usually do all the work in the main playground file, then when I am done testing I move it to functions in here which could be imported and used for testing